server/stream.go

```diff
@@ -366,10 +366,21 @@ func (a *Account) addStreamWithAssignment(config *StreamConfig, fsConfig *FileSt
 		return nil, ApiErrors[JSStreamReplicasNotSupportedErr]
 	}
 
+	// Make sure we are ok when these are done in parallel.
+	var wg sync.WaitGroup
+	v, loaded := jsa.inflight.LoadOrStore(cfg.Name, &wg)
```

Comment:
Minor nit but `LoadOrStore(cfg.Name, &sync.WaitGroup{})` is clearer and doesn't pollute the scope with extra names.

Reply:
Good call! Will update.

Reply:
Done.